### ISSUE TITLE
long wait time after entering password in 0.7.1

### ISSUE THREAD

**Issue by hossamdash** [apognu/tuigreet#40]
my screen shows a blinking "-" symbol for about 10 seconds after entering password and pressing enter. downgrading to 0.6.1 solves the problem. here's my /etc/greetd/config.toml for reference. I'm using the [package](https://aur.archlinux.org/packages/greetd-tuigreet/) from AUR
```
[terminal]
# The VT to run the greeter on. Can be "next", "current" or a number
# designating the VT.
vt = 1

# The default session, also known as the greeter.
[default_session]

# `agreety` is the bundled agetty/login-lookalike. You can replace `$SHELL`
# with whatever you want started, such as `sway`.
command = "tuigreet --issue --time --remember --remember-session"

# The user to run the command as. The privileges this user must have depends
# on the greeter. A graphical greeter may for example require the user to be
# in the `video` group.
user = "greeter"
```

**Comment by apognu** [comment]
@hossamdash Are you able to give one of [these artifacts](https://github.com/apognu/tuigreet/actions/runs/1329862774) a try? The issue seems to be fixed on my side, but I'd like to give it some testing before merging it into master.

**Comment by hossamdash** [comment]
@apognu I tried your latest dev branch [artifacts](https://github.com/apognu/tuigreet/actions/runs/1330337525) and the issue seems to be fixed, now the time is more like 1.8 seconds which is similar to what i was getting with [ly](https://github.com/nullgemm/ly). thanks for solving the issue so quick and for the awesome work with this tool :D.

**Comment by apognu** [comment]
Yeah, from my tests, the leftover delay might actually come from `greetd` and the initialization time of whatever you run.

I'll do more testing tomorrow and keep using this version for a while before I can be confident merging it into master.

I'll close this when it is merged.
